### PR TITLE
docs: add some tips on collecting per-queue metrics for RabbitMQ

### PR DESCRIPTION
**Pull request by HG00**
##### Summary
Added some tips on collecting per-queue metrics for RabbitMQ. I had some struggles getting per-queue rmq set up, just adding some details to the permissions needed that I found unintuitive on the rmq permissions side.

### CODE REVIEW

File: collectors/python.d.plugin/rabbitmq/README.md [hunk 1]
```diff
@@ -113,4 +113,11 @@ socket:
 
 ---
 
+### Per-Queue Chart configuration
+RabbitMQ users with the "monitoring" tag cannot see all queue data, you'll need a user with read permissions here. To create a dedicated user for netdata:
```
Comment (ilyam8) [inline collectors/python.d.plugin/rabbitmq/README.md:117]:
```suggestion
### Per-Queue Chart configuration

RabbitMQ users with the "monitoring" tag cannot see all queue data, you'll need a user with read permissions here. To create a dedicated user for netdata:
```

File: collectors/python.d.plugin/rabbitmq/README.md [hunk 2]
```diff
@@ -113,4 +113,11 @@ socket:
...
 
+```bash
+rabbitmqctl add_user netdata ChangeThisSuperSecretPassword
+rabbitmqctl set_permissions netdata "^$" "^$" ".*"
+```
+Once the user is set up, add `collect_queues_metrics: yes` to your config.
```
Comment (ilyam8) [inline collectors/python.d.plugin/rabbitmq/README.md:123]: Let's add an actual example with user/pass/collect_queues_metrics

Comment (HG00) [reply collectors/python.d.plugin/rabbitmq/README.md]: good point, added :)

File: collectors/python.d.plugin/rabbitmq/README.md (hunk 1 above)
Comment (ilyam8) [reply collectors/python.d.plugin/rabbitmq/README.md:117]: an empty line after the header, please

**Review by ilyam8 — APPROVED** [review]

File: collectors/python.d.plugin/rabbitmq/README.md [hunk 3]
```diff
@@ -113,4 +113,22 @@ socket:
 
 ---
 
+### Per-Queue Chart configuration
 
+RabbitMQ users with the "monitoring" tag cannot see all queue data, you'll need a user with read permissions here. To create a dedicated user for netdata:
+
```
Comment (kickoke) [inline collectors/python.d.plugin/rabbitmq/README.md:119]:
```suggestion
RabbitMQ users with the "monitoring" tag cannot see all queue data. You'll need a user with read permissions. 
To create a dedicated user for netdata:

```

File: collectors/python.d.plugin/rabbitmq/README.md [hunk 4]
```diff
@@ -113,4 +113,22 @@ socket:
...
+```bash
+rabbitmqctl add_user netdata ChangeThisSuperSecretPassword
+rabbitmqctl set_permissions netdata "^$" "^$" ".*"
+```
```
Comment (kickoke) [inline collectors/python.d.plugin/rabbitmq/README.md:123]: Could you give a short explanation what these two lines will do?

File: collectors/python.d.plugin/rabbitmq/README.md [hunk 5]
```diff
@@ -113,4 +113,22 @@ socket:
...
+Once the user is set up, add `collect_queues_metrics: yes` to your config:
```
Comment (kickoke) [inline collectors/python.d.plugin/rabbitmq/README.md:124]: To the RabbitMQ config or the Netdata config?

**Review by kickoke — COMMENTED** [review]
Hi HG00! 
Thanks for your help with the docs. 

I left some comments to make the content even clearer. 

Kind regards, 

Tina

Technical writer at Netdata

File: collectors/python.d.plugin/rabbitmq/README.md [hunk 6]
```diff
@@ -113,4 +113,23 @@ socket:
...
+### Per-Queue Chart configuration
 
+RabbitMQ users with the "monitoring" tag cannot see all queue data. You'll need a user with read permissions. 
+To create a dedicated user for netdata:
+
+```bash
+rabbitmqctl add_user netdata ChangeThisSuperSecretPassword
+rabbitmqctl set_permissions netdata "^$" "^$" ".*"
+```
+Once the user is set up, add `collect_queues_metrics: yes` to your config:
```
Comment (ilyam8) [inline collectors/python.d.plugin/rabbitmq/README.md:125]:
```suggestion
Once the user is set up, add `collect_queues_metrics: yes` to your `rabbitmq.conf`:
```

File: collectors/python.d.plugin/rabbitmq/README.md [hunk 7]
```diff
@@ -113,4 +113,23 @@ socket:
...
 
+### Per-Queue Chart configuration
 
+RabbitMQ users with the "monitoring" tag cannot see all queue data. You'll need a user with read permissions. 
+To create a dedicated user for netdata:
+
+```bash
+rabbitmqctl add_user netdata ChangeThisSuperSecretPassword
+rabbitmqctl set_permissions netdata "^$" "^$" ".*"
+```
```
Comment (ilyam8) [inline collectors/python.d.plugin/rabbitmq/README.md:124]:
```suggestion
```

See [set_permissions](https://www.rabbitmq.com/rabbitmqctl.8.html#set_permissions) for details.

```

File: collectors/python.d.plugin/rabbitmq/README.md [hunk 8]
```diff
@@ -113,4 +113,22 @@ socket:
...
 
+### Per-Queue Chart configuration
 
+RabbitMQ users with the "monitoring" tag cannot see all queue data. You'll need a user with read permissions. 
+To create a dedicated user for netdata:
+
+```bash
+rabbitmqctl add_user netdata ChangeThisSuperSecretPassword
+rabbitmqctl set_permissions netdata "^$" "^$" ".*"
+Once the user is set up, add `collect_queues_metrics: yes` to your `rabbitmq.conf`:
```
Comment (ilyam8) [inline collectors/python.d.plugin/rabbitmq/README.md:124]:
```suggestion
```bash
rabbitmqctl add_user netdata ChangeThisSuperSecretPassword
rabbitmqctl set_permissions netdata "^$" "^$" ".*"
```

See [set_permissions](https://www.rabbitmq.com/rabbitmqctl.8.html#set_permissions) for details.

Once the user is set up, add `collect_queues_metrics: yes` to your `rabbitmq.conf`:
```

**Review by ilyam8 — APPROVED** [review]

**Review by ilyam8 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**